docs/includes/fundamentals/read-operations/ReadOperationsTest.php

```diff
@@ -94,4 +101,35 @@ public function testFirst(): void
         $this->assertNotNull($movie);
         $this->assertInstanceOf(Movie::class, $movie);
     }
+
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testText(): void
+    {
+        // start-text
+        $movies = Movie::where('$text', ['$search' => '"love story"'])
```

Comment:
Test failing, the result is 1: only the movie with the title "this is a love story" is matched. We get 2 results by removing the quotes.

```suggestion
        $movies = Movie::where('$text', ['$search' => 'love story'])
```